cogs/boards.py

```diff
@@ -123,10 +123,15 @@ async def board_loop(self):
                 if "mon_today" in board['type']:
                     mon_today = await queries.statboard_mon_today(self.bot.config, area[0])
                     if "mon_active" in board['type']:
-                        text = f"{text}{self.bot.custom_emotes['pokeball']} **{mon_active[0][0]:,}** {self.bot.locale['active_pokemon']} | **{mon_today[0][0]:,}** {self.bot.locale['today']}\n\n"
+                        scanned_ratio = int(round((mon_today[0][1] / mon_today[0][0] * 100), 0))
+                        text = f"{text}{self.bot.custom_emotes['pokeball']} **{mon_active[0][0]:,}** {self.bot.locale['active_pokemon']} | **{mon_today[0][0]:,}** ({scanned_ratio}% mit IVs) {self.bot.locale['today']}\n\n"

```

Comment:
- This needs translations for "mit" right now it's just German
- If you're implementing it for mon_today, then it should also show for mon_active and mon_today without mon_active enabled
- I think you forgot to adjust the query?
- Querying for mons with IVs takes ages in my big mon table. The Pokemon query itself already eats my CPU,  this might be too much now. This should be an extra option